XiEditor/EditView.swift

```diff
@@ -56,6 +56,45 @@ struct TextDrawingMetrics {
     }
 }
 
+/// The pre-renderered cache of the digits 0-9 for display in the gutter.
+/// These can be scatter-gathered to quickly construct any sequence of digits.
+struct GutterCache {
+    private let font: CTFont
+    // Scatter-gather from here for the gutter for lines not containing a cursor.
+    private let noCursorCache: TextLine
+    // Scatter-gather from here for the gutter for lines containing a cursor.
+    private let cursorCache: TextLine
+
+    // Initializes the cache with the given font & the requested colors for when
```

Comment:
`///` here would make this a doc-comment, rendered on hover in Xcode. I regularly find myself checking this reference: http://nshipster.com/swift-documentation/

Reply:
Fixed.